docs/data.md

```diff
@@ -1,40 +1,59 @@
+Global numerical weather prediction (NWP) models are used to generate medium range forecasts and provide the initial and boundary conditions for higher-resolution regional models. Their global coverage makes them a good starting point for demonstrating the application of scoring methods in any region of interest.
 
-For more information see https://www.ecmwf.int/en/forecasts/datasets/reanalysis-datasets/era5 and
-https://github.com/pangeo-data/WeatherBench
+The Bureau of Meteorology provides global model data from the ACCESS NWP system. See [https://doi.org/10.25914/608a993391647](https://doi.org/10.25914/608a993391647) for more information.
 
-## Gridded satellite (observation) data
-Satellite data varies according to the type of orbit and purpose of the mission. It is too complex to quickly address in a demonstration. A guide on working with satellite data may be added in future.
+Global model data is also available from the National Oceanic and Atmospheric Administration (NOAA) Global Forecast System (GFS). See [https://www.ncei.noaa.gov/products/weather-climate-models/global-forecast](https://www.ncei.noaa.gov/products/weather-climate-models/global-forecast) for more information.
 
-## Gridded radar (observation) data
-Radar data also varies according to region and is not a globally standardised data set. Information on Australian based radars can be found at https://www.openradar.io/
+#### Point-Based Data
+
+Point-based observations are shared routinely between countries for the purposes of weather modelling.
+
+The NOAA Integrated Surface Database (ISD) provides hourly point-based (aka in-situ) weather station data globally. It is a good starting point for understanding how to work with point-based data. For more information about the NOAA ISD see [https://www.ncei.noaa.gov/products/land-based-station/integrated-surface-database](https://www.ncei.noaa.gov/products/land-based-station/integrated-surface-database).
+
+#### Gridded Model Reanalysis Data
+
+Reanalysis datsets typically span years if not decades. 
+
+The ECMWF Reanalysis v5 (ERA5) dataset is a well known and widely used global reanalysis dataset. For more information see [https://www.ecmwf.int/en/forecasts/dataset/ecmwf-reanalysis-v5](https://www.ecmwf.int/en/forecasts/dataset/ecmwf-reanalysis-v5). ERA5 is also incluced in [WeatherBench 2](https://sites.research.google/weatherbench/).
+
+#### Gridded Radar (Observation) Data
+
+Radar data varies according to region and is not a globally standardised data set. Information on Australian based radars can be found at [https://www.openradar.io/](https://www.openradar.io/).
+
+### Software for Acessing Data
+
+#### CliMetLab
+
+The European Centre for Medium-Range Weather Forecasts (ECMWF) has developed the CliMetLab Python package to simplify access to a large range of climatological and meteorological datasets. See [https://climetlab.readthedocs.io/](https://climetlab.readthedocs.io/).
+
```

Comment:
I also wasn't sure whether WeatherBench 2 should also be added to the "Software for Accessing Data" section.